pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -204,6 +209,17 @@ class _Parser {
       }
     }
 
+    var testRandomizeOrderingSeed =
+        _parseOption('test-randomize-ordering-seed', (value) {
+      var seed = value == 'random'
+          ? Random().nextInt(4294967295)
+          : int.parse(value).toUnsigned(32);
+      if (seed != null && seed > 0) {
+        print("Shuffling test order with --test-randomize-ordering-seed=$seed");
```

Comment:
@grouma I am printing the seed here. I can use some other logging mechanism than `print` if you want.

Reply:
I missed this print. 

[nit] Use single quotes for consistency.

I think printing here is fine but @natebosch may have thoughts on if this should use the reporter API in some fashion.

Reply:
I don't think we can use the `print` when a client is using the json reporter.

Why do we need to print this? I would think the logging outside of the test package would remember the seed.

Reply:
Discussed offline. We can add it to the report when someone requests it but for now what we have is fine. As for why it is required, someone can provide `random` and therefore we need to log the seed that was used.